notebooks/Model_Training_with_BERT.ipynb

```diff
@@ -1420,7 +1420,7 @@
        "      <td>&lt;NA&gt;</td>\n",
```

Comment:
<p>hmm, any idea why you got more true positives than me, just random?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>Yes, the algorithm used is literally random. Well, randomized. I haven't found a way to fix the random seeds used.</p>